src/core/Board.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { Card } from "common/components";
+import { CardFaces } from "common/components/CardFaces";
+
+import styles from "./Board.module.scss";
+
+export const Board = () => {
+  const doubleCards = arr => {
+    return arr.concat(arr);
+  };
+
+  //https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array
+  const shuffle = arr => {
+    let currentIndex = arr.length;
+    let temporaryValue = 0;
+    let randomIndex = 0;
+
+    while (0 !== currentIndex) {
+      randomIndex = Math.floor(Math.random() * currentIndex);
+      currentIndex -= 1;
+
+      temporaryValue = arr[currentIndex];
+      arr[currentIndex] = arr[randomIndex];
+      arr[randomIndex] = temporaryValue;
+    }
+
+    return arr;
+  };
+
+  const randCards = shuffle(doubleCards(CardFaces));
```

Comment:
For readability, there's usually no harm in spelling out words in full. I recommend renaming this variable to `randomCards`, `randomizedCards`, or even `shuffledCards` :)